### PR TITLE
correctly capitalize log lines

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

/kind cleanup
#### What this PR does / why we need it:
This PR capitalizes the first letter of every log line. This helps developers grep for where a log line came from. It also adds a make target to verify new PRs also capitalize the first letter correctly.

a gift for you @saschagrunert :heart: 
#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -137,6 +137,11 @@ lint: .gopathok ${GOLANGCI_LINT}
 	${GOLANGCI_LINT} linters
 	${GOLANGCI_LINT} run
 
+check-log-lines:
+	./hack/log-capitalized.sh
+	./hack/tree_status.sh
+
```
Comment (saschagrunert) [inline Makefile:143]:
```suggestion
```

File: .github/workflows/verify.yml [hunk 2]
```diff
@@ -53,6 +53,20 @@ jobs:
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+      - uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: go-${{ hashFiles('**/go.sum') }}
+          restore-keys: go-
+      - run: make check-vendor
```
Comment (saschagrunert) [inline .github/workflows/verify.yml:68]:
```suggestion
      - run: make check-log-lines
```

**Comment by saschagrunert** [comment]
Let's remove this manipulation from the hook:
https://github.com/cri-o/cri-o/blob/c3ae331efa2e27f938d919ba27b1324bf765ecaf/internal/log/hook_filter.go#L58-L63

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by saschagrunert** [comment]
Oh:
```
internal/log/hook_filter.go:6:2: imported and not used: "unicode"
```

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

**Comment by saschagrunert** [comment]
Unfortunately this needs a rebase.

**Comment by haircommander** [comment]
rebased!

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**